From f5158d68103922433b5aa6a26a543ace34c932a6 Mon Sep 17 00:00:00 2001
From: Oliver Brown <oliver.brown@nxp.com>
Date: Thu, 2 Aug 2018 07:51:26 -0500
Subject: [PATCH 4332/5242] MLK-19119 arm64: dts: imx8qm: Correct bus clock
 for HDMI Interrupt Steer

commit  4a3f9d071a34c4ddf035364eb685306e6cd83e4f from
https://source.codeaurora.org/external/imx/linux-imx.git

The Local Interrupt Steer Clock Bus Clock (LIS IPG) should be 83.375 MHz.

Signed-off-by: Oliver Brown <oliver.brown@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index be75f8c..64e703b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -2707,7 +2707,7 @@
 		clock-names = "ipg";
 		assigned-clocks = <&clk IMX8QM_HDMI_DIG_PLL_CLK>,
 							<&clk IMX8QM_HDMI_LIS_IPG_CLK>;
-		assigned-clock-rates = <675000000>, <85000000>;
+		assigned-clock-rates = <675000000>, <84375000>;
 		power-domains = <&pd_hdmi>;
 	};
 
-- 
1.7.9.5

